pkg/lhutil/lighthouse.go

```diff
@@ -100,3 +114,8 @@ func GetServiceExportCondition(status *mcsv1a1.ServiceExportStatus, ct mcsv1a1.S
 
 	return latestCond
 }
+
+func ServiceExportConditionEqual(c1, c2 *mcsv1a1.ServiceExportCondition) bool {
```

Comment:
This is a function that is currently in the agent code, moved it to the util to reuse it in my controller

Reply:
that's fine. The append condition function could also be useful in `lhutil`

Reply:
Ok, I added it also to `lhutil`